README.md

```diff
@@ -62,6 +62,9 @@ Fields:
  - `enabledFeatures`. Type: \[String\]. Optional. Default value: empty array. Used on the front end to determine which opt-in features the institution has access to. (Either make this required, or make `disabledFeatues` optional. If this field is left optional, remove default value.)
  - `facebookId`: Type: String. Optional. The id of the facebook page using our facebook app that is associated with the institution.
  - `filterOn` Type: Boolean. Optional. The field that distinguished open bots (value `false`) from closed bots (value `true`). (To do: move this to `dialogSettings`. Make required.)
+ - `freeze` Type: Object. Optional. Enable and control "freezing" the bot when an admin sends a message.  See AH-1054
+   - `enable` Type: Boolean.  If true, enable freezing.
```

Comment:
A thought about this: there is already an object somewhere on this document that is supposed to turn features on and off. Would it be worth having the thing that enables the feature in there? Genuine question. I'm not sure on the best way to group fields here.

Reply:
I may just be applying previous experience, but my impression was that that was intended more for short-term enabling/disabling test features.  In my mind, for something like this where there is configuration, it makes sense to batch enabled/disabled with the configuration.  That's not a strong opinion, though; it's just my default.